src/Command/ImportQuestionnairesCommand.php

```diff
@@ -0,0 +1,291 @@
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImportQuestionnairesCommand extends StatisticsCommand
+{
+
+    /**
+     * @var EntityManagerInterface $em
+     */
+    private $em;
+
+    /**
+     * @var QuestionnaireRepository $questionnaireRepo
+     */
+    private $questionnaireRepo;
+
+    /**
+     * @var AspectRepository $aspectRepo
+     */
+    private $aspectRepo;
+
+    /**
+     * @var QuestionRepository $questionRepo
+     */
+    private $questionRepo;
+
+    /**
+     * @var HelpRepository $helpRepo
+     */
+    private $helpRepo;
+
+    /**
+     * @var string $pathToJson
+     */
+    private $pathToJson = "imports/questionnaire_imports.json";
+
+
+    /**
+     * @param EntityManagerInterface $em
+     * @param QuestionnaireRepository $questionnaireRepo
+     * @param AspectRepository $aspectRepo
+     * @param QuestionRepository $questionRepo
+     * @param HelpRepository $helpRepo
+     */
+    public function __construct(
+        EntityManagerInterface $em,
+        QuestionnaireRepository $questionnaireRepo,
+        AspectRepository $aspectRepo,
+        QuestionRepository $questionRepo,
+        HelpRepository $helpRepo
+    ) {
+        parent::__construct();
+
+        $this->em = $em;
+        $this->questionnaireRepo = $questionnaireRepo;
+        $this->aspectRepo = $aspectRepo;
+        $this->questionRepo = $questionRepo;
+        $this->helpRepo = $helpRepo;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName("app:import-questionnaire");
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->writeln("Starting import of questionnaires...");
+
+        if (!file_exists($this->pathToJson)) {
+            $output->writeln("No data to import. File at " . $this->pathToJson . " not found.");
+            return 1;
+        }
+
+        $questionnaires = JsonMachine::fromFile($this->pathToJson);
+        foreach ($questionnaires as $questionnaire) {
+            $this->importQuestionnaire($questionnaire);
+        }
+
+        $output->writeln("Questionnaire import process has finished.");
+        return 0;
+    }
+
+    /**
+     * @param $questionnaire
+     */
+    private function importQuestionnaire($questionnaire): void
+    {
+        $db_questionnaire = $this->questionnaireRepo->findOneBy(["type" => $questionnaire["type"]]);
+
+        if (!$db_questionnaire) {
+            $db_questionnaire = new Questionnaire();
+            $db_questionnaire->setType($questionnaire["type"]);
+        }
+
+        $aspects = $questionnaire["aspects"];
+
+        $this->em->persist($db_questionnaire);
+        $this->em->flush();
+
+        foreach ($aspects as $aspect) {
+            $isDeprecated = false;
+            if (array_key_exists("deprecated", $aspect)) {
+                $isDeprecated = $aspect["deprecated"];
+            }
```

Comment:
This can be problematic `array_key_exists` will return true if the key "deprecated" exists but if it is NULL it will still return true because the key exists and therefore `$isDepracted` will end up being NULL.

Instead you could do the following: `$isDeprecated = isset($aspect["deprecated"]) ? $aspect["deprecated"] : false;`
This will make sure that not only the key "deprecated" exists but that the value is not NULL.